components/renku_data_services/migrations/utils.py

```diff
@@ -0,0 +1,139 @@
+"""Custom migrations env file to support modular migrations."""
```

Comment:
This whole file is just the former content of env.py with minor changes.

For a true diff run git diff main:components/renku_data_services/migrations/env.py HEAD:components/renku_data_services/migrations/utils.py` after you have checked out this branch.